doc/API.md

```diff
@@ -51,18 +51,29 @@ The Datadog tracing module defines the following configuration directives.
 ### `datadog_service_name`
 - **syntax** `datadog_service_name <name>`
 - **default**: `nginx`
-- **context**: `http`
+- **context**: `http`, `server`
```

Comment:
is there a reason to not add the directive in location block ? I will add this question once but it is the same for the others directives

Reply:
addressed in b558dc79273d05d6ab4a26f008a79a1c7d78c812